tensorly/backend/jax_backend.py

```diff
@@ -98,14 +105,5 @@ def logsumexp(tensor, axis=0):
 for name in ["solve", "qr", "svd", "eigh"]:
     JaxBackend.register_method(name, getattr(np.linalg, name))
 
-if Version(jax.__version__) >= Version("0.3.0"):
-
-    def index_update(tensor, indices, values):
-        return tensor.at[indices].set(values)
-
-    JaxBackend.register_method("index_update", index_update)
-else:
-    JaxBackend.register_method(name, getattr(jax.ops, name))
```

Comment:
I'm pretty sure the `else` statement just never worked here? It looks like `name` is undefined.

Reply:
Oh dear - you're right. That change was early days of Jax